### ISSUE TITLE
In correct Hyper Link for Best Practices Analyzer for Web Application Proxy

### ISSUE THREAD

**Issue by Nandeeshswami** [MicrosoftDocs/windowsserverdocs#5565]
[Enter feedback here]
The hyperlink for Best Practices Analyzer for Web Application Proxy is broken. Can you please help fix?

Here is the right link,
https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/dn383651(v=ws.11)

Current link,
assetid:///995f5cea-e334-4ce6-a14c-6a2d03c0c79a

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d126ae85-fc95-be1c-faeb-90b9279f734a
* Version Independent ID: 167f764c-bc47-25a7-b31b-7fdf18c81749
* Content: [Troubleshooting Web Application Proxy](https://docs.microsoft.com/en-us/windows-server/remote/remote-access/web-application-proxy/troubleshooting-web-application-proxy)
* Content Source: [WindowsServerDocs/remote/remote-access/web-application-proxy/Troubleshooting-Web-Application-Proxy.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/remote/remote-access/web-application-proxy/Troubleshooting-Web-Application-Proxy.md)
* Product: **windows-server**
* Technology: **remote**
* GitHub Login: @JasonGerend
* Microsoft Alias: **kgremban**

**Comment by illfated** [comment]
Sure thing. Thank you for reporting the outdated link format and the correct link suggestion! 👍🏼

**Comment by illfated** [comment]
By the way; seeing as you have provided a link to the `/previous-versions/` archive, I presume you have tried to locate any current documentation containing updated material for the same topic, but could not find one. I also presume that this content may be missing in current documentation or might be nearing discontinuation.

**Comment by Nandeeshswami** [comment]
That's right. If there is no right version, then should we remove the reference altogether.

**Comment by illfated** [comment]
Thank you for confirming. I will start working on a PR to replace the archaic internal link (from the time before GitHub) with a functional one.

**Comment by illfated** [comment]
There you go. PR #5578 is now available for review, comments, and suggestions.

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
